### ISSUE TITLE
ssh not installed - unable to run check_by_ssh (returned status 3)

### ISSUE THREAD

**Issue by tr1plus** [manios/docker-nagios#27]
Hi

The check_by_ssh plugin only started working after I manually re-added the openssh package

```
/opt/nagios # apk update

fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/main/x86_64/APKINDEX.tar.gz

fetch http://dl-cdn.alpinelinux.org/alpine/v3.11/community/x86_64/APKINDEX.tar.gz

v3.11.11-26-gaa55c5a654 [http://dl-cdn.alpinelinux.org/alpine/v3.11/main]

v3.11.11-24-g127ddb5cee [http://dl-cdn.alpinelinux.org/alpine/v3.11/community]

OK: 11274 distinct packages available

/opt/nagios # apk add openssh

(1/6) Installing openssh-keygen (8.1_p1-r0)

(2/6) Installing openssh-client (8.1_p1-r0)

(3/6) Installing openssh-sftp-server (8.1_p1-r0)

(4/6) Installing openssh-server-common (8.1_p1-r0)

(5/6) Installing openssh-server (8.1_p1-r0)

(6/6) Installing openssh (8.1_p1-r0)
```

I see in the dockerfile you are adding the openssh-client, but it doesn't seem to be enough (as ssh wasn't found)

**Comment by manios** [comment]
Hi @tr1plus !

Thanks for the finding! Indeed it is missing and `check_by_ssh` does not work. I will add the openssh package it in the next version.

Best regards,  
Christos

**Comment by manios** [comment]
Hi @tr1plus !

I have released a new `latest` tag based on `alpine:3.14` with the appropriate changes. Can you please give it a try

```shell
docker pull manios/nagios:latest
# or
docker pull manios/nagios:4.4.6
```

and provide me with feedback ? I am closing the issue. If you find any discrepancy please post a comment here and we will take a look.

Thanks,
Christos

**Comment by tr1plus** [comment]
seems to be working! Thanks
